hack/make/build-integration-test-binary

```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env bash
-# required by https://github.com/AkihiroSuda/kube-moby-integration
```

Comment:
@AkihiroSuda https://github.com/AkihiroSuda/kube-moby-integration is archived. LGTY to remove this file?

Reply:
LGTY